reference/callback-functions.md

```diff
@@ -88,24 +88,24 @@ Since Webots 7.2.0, a multi-threaded version of ODE is used.
 Therefore, this function may be called from different threads.
 You should ensure it is re-entrant and that every ODE function call modifying the ODE world is protected by mutexes as explained earlier.
 
-### void webots\_physics\_step()
+### "void webots\_physics\_step()"
```

Comment:
Didn't we decide to not use the parenthesis for referring to a function?

Reply:
In this case it's not obvious, because later in this file, we have similar titles but with arguments.